core/src/main/scala/org/apache/spark/deploy/rest/SubmitRestProtocolResponse.scala

```diff
@@ -17,14 +17,14 @@
 
 package org.apache.spark.deploy.rest
 
-import java.lang.Boolean
+import java.lang.{Boolean => JBoolean}
```

Comment:
I think this is fine. We have different practice too, e.g., in sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/literals.scala